Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -96,7 +99,16 @@ void Awake()
 
 	void Update()
 	{
-		daylightIntensity = Mathf.Cos( DayCycleManager.dayCycleTimer / DayCycleManager.dayCycleLength * 2.0f * Mathf.PI ) * -0.5f + 0.5f;
+		float timeOfDay = DayCycleManager.currentTime / DayCycleManager.dayCycleLength;
```

Comment:
I would put the night fraction adjustment in here.

    timeOfDay = DayCycleManager.currentTime / ( DayCycleManager.dayCycleLength * ( 1.0f - _nightFraction) );

Then you compare it to 1.0f in the if statement and take out the division from the cosine line